README.md

```diff
@@ -209,6 +211,18 @@ Such valuable information includes a stack trace of the crash. Here's a quick st
 
 If you want to remove lldb from your debug setup again you can simply run `rm ~/.mtouch-launch-with-lldb` in your Terminal.
 
+
+#### Simulator
+
+1. Launch your app in debug mode in Visual Studio for Mac.
+2. Open your Activity Monitor app.
+3. In the Activity Monitor app search for your app name in the search bar. To give an example, if I want to attach to our PSPDFCatalog example app I need to search for "PSPDFCatalog".
+4. Double-Click on the process to view the information window.
+5. Now you need to identify the PID (Process ID) of your process. The PID is the number included in the round brackets in the window title, e.g. if the title says "PSPDFCatalog (73389)", then 73389 is your PID. Here's an example of how that looks: <img width="50%" src="Images/pid.png"/>
```

Comment:
I think the image should be on a new line. Here's how it looks now:

<img width="859" alt="Screen Shot 2021-01-13 at 2 26 21 PM" src="https://user-images.githubusercontent.com/7443038/104499656-69ef6f80-55ab-11eb-9827-aa4da44d443a.png">


```suggestion
5. Now you need to identify the PID (Process ID) of your process. The PID is the number included in the round brackets in the window title, e.g. if the title says "PSPDFCatalog (73389)", then 73389 is your PID. Here's an example of how that looks: 
<img width="50%" src="Images/pid.png"/>
```

Reply:
Ah yeah I tried that as well but this results in points 6,7, and 8 being in a single line 🤦‍♂️  Will fix this and merged afterwards, thanks for catching that!